src/platform/datapath_winuser.c

```diff
@@ -734,6 +741,28 @@ CxPlatDataPathQuerySockoptSupport(
         Datapath->Features |= CXPLAT_DATAPATH_FEATURE_RECV_COALESCING;
     }
 }
+    //
+    // TODO: This "TTL_FEATURE check" code works, and mirrors the approach for Kernel mode.
+    //       However, it is considered a "hack" and we should determine whether or not
+    //       the current release story fits this current workaround.
+    //
+    HMODULE NtDllHandle = LoadLibraryA("ntdll.dll");
```

Comment:
We shouldn't have this logic in two places. Perhaps, instead we update `CX_PLATFORM` to store the `BuildNumber` (on Windows) and put this logic in platform_winuser.c (and might as well update _winkernel.c).